CHANGELOG.md

```diff
@@ -1,5 +1,21 @@
 # Changelog
 
+## [1.24](https://github.com/brave/brave-ios/releases/tag/v1.24)
+
+- Implemented opt-in for Brave Today. ([#3374](https://github.com/brave/brave-ios/issues/3374))
+- Added support for OpenSearch. ([#936](https://github.com/brave/brave-ios/issues/936))
+- Added server location support for Brave Firewall + VPN. ([#2934](https://github.com/brave/brave-ios/issues/2934))
+- Added support to remove other devices from sync chain. ([#3195](https://github.com/brave/brave-ios/issues/3195))
+- Added support for custom RSS feeds within Brave Today. ([#3249](https://github.com/brave/brave-ios/issues/3249))
+- Added support for Japanese content feed of Brave Today. ([#3051](https://github.com/brave/brave-ios/issues/3051))
+- Added share button in Brave Shields to increase app virality. ([#2981](https://github.com/brave/brave-ios/issues/2981))
+- Added support for including New Tab Page Sponsored Image views in “Estimated pending rewards” and “Ad notifications received this month” for Brave Ads. ([#3235](https://github.com/brave/brave-ios/issues/3235))
+- Added ability to control “Allow universal links to open in external apps” via “Other Privacy Settings”. ([#2119](https://github.com/brave/brave-ios/issues/2119))
+- Improved visuals for YubiKey security key PIN modal. ([#2129](https://github.com/brave/brave-ios/issues/2129))
+- Improved Brave Today by updating feed when publishers are removed or added. ([#3033](https://github.com/brave/brave-ios/issues/3033))
+- Updated New Tab Page by moving “Show More” favorites button up 20px. ([#3359](https://github.com/brave/brave-ios/issues/3359))
```

Comment:
Can remove this one from changelog. Its very trivial that users probably wont even notice. Other than that everything else looks good

Reply:
Was debating about keeping it or not haha. I can remove it though.